ocaml/src/albamgr_test.ml

```diff
@@ -20,9 +20,9 @@ open Albamgr_protocol.Protocol
 
 let host = "127.0.0.1"
 let hosts = [ host ]
-let ccfg_ref = ref None
+let ccfg_url_ref = ref None
 
-let get_ccfg () = Option.get_some (!ccfg_ref)
+let get_ccfg_url () = Option.get_some (!ccfg_url_ref)
```

Comment:
It probably would've been easier (required less changes) if this type had stayed `unit -> Albamgr_protocol.Protocol.Arakoon_config.t` iso changing to `unit -> Prelude.Url.t`?